src/runtime-rs/crates/hypervisor/src/ch/inner_hypervisor.rs

```diff
@@ -229,7 +229,13 @@ impl CloudHypervisorInner {
     async fn cloud_hypervisor_launch(&mut self, _timeout_secs: i32) -> Result<()> {
         self.cloud_hypervisor_ensure_not_launched().await?;
 
-        let debug = false;
+        let cfg = self
+            .config
+            .as_ref()
+            .ok_or("no hypervisor config for CH")
+            .map_err(|e| anyhow!(e))?;
```

Comment:
Do we want a hard error for a missing config, or wouldn't that be a case where you would want to set `debug` to `true`?

Reply:
@c3d - That's a good question but that code is used a few times and is doing "double time" as here we want to know:

- That this API was called after we set the generic hypervisor config passed to us by runtime-rs (in `set_hypervisor_config()`). 
- If debug was requested.

I don't like the duplication but I'm being paranoid to protect against runtime-rs calling the hypervisor methods in the wrong order.